docs/calico-with-docker/docker-network-plugin/CalicoAsService.md

```diff
@@ -0,0 +1,63 @@
+<!--- master only -->
+> ![warning](../../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.14.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Calico as a Docker network plugin service
+
+In some cases, you may want to run a calico docker image as a service.
+You can run the `calico/node-libnetwork` image as a standalone container 
+service if you bypass the `calicoctl` command that starts the libnetwork
+node.
+
+## Start Calico's libnetwork Node with Docker
+For this example, you can run the image using a `docker run` command:
+
+```
+docker run -d --privileged --net=host \
+    -v /run/docker/plugins:/run/docker/plugins \
+    --name=calico-libnetwork calico/node-libnetwork
+```
+
+In this example,
+
+ - Privileged is required since the container creates network devices.
+ - Host network is used since the network changes need to occur in the host namespace
+ - The /run/docker/plugins volume is used to allow the plugin to communicate with Docker.
+
+If you don't have etcd available at localhost:4001 then you need to pass 
+in the location as an environment variable e.g. `-e ETCD_AUTHORITY=1.2.3.4:2379`.
+
+## Sample Systemd Config
+Here is an example config that you could use as your calico-libnetwork.service job for running the calico-libnetwork docker container with systemd.
+
+```
+[Unit]
+Description=calico-libnetwork
+After=docker.service
+Requires=docker.service
```

Comment:
Turk mentioned to me that depending on how Docker is installed this might be docker or dockerengine.... touch base with him.

I don't think this needs changing, but it might be nice to point this out below as a note.